reference/tools/tiup/playground.md

```diff
@@ -0,0 +1,110 @@
+
+- The version is not specified. In this situation, TiUP uses the latest version of the installed playground component. Assume that the latest version is v0.0.6, this command will work the same as `tiup playground:v0.0.6`.
+- If you have not used TiUP to install the TiDB/TiKV/PD component, the playground component uses the latest stable version of these components, and then these instances are started and operated.
+- The version of each TiDB/PD/TiKV component is not specified. By default, the latest released version of each component is used. Assume that the latest version is v4.0.0-rc, this command will work the same as `tiup playground:v0.0.6 v4.0.0-rc`.
+- The playground component does not specify the number of each component. By default, a cluster that consists of one TiDB instance, one TiKV instance, and one PD instance is started.
+- After starting each component, the playground reminds you that the cluster is successfully started and provides you some useful information, such as how to connect to the TiDB cluster through the MySQL client and how to access the dashboard.
+
+The flags of the playground command-line tool are described as follows:
+
+```bash
+Flags:
+      --db int                   Specifies the number of TiDB instances in the cluster, 1 by default.
+      --db.binpath string        Specifies the location of TiDB binary files, for development and debugging. You can ignore this flag.
+      --db.config string         Specifies TiDB configuration files, for development and debugging. You can ignore this flag.
+  -h, --help                     Prints the help information.
+      --host string              Sets the listening address of each component (127.0.0.1 by default). For other computers to access components, you can set the address to 0.0.0.0.
+      --kv int                   Specifies the number of TiKV instances in the cluster, 1 by default.
+      --kv.binpath string        Specifies the location of TiKV binary files, for development and debugging. You can ignore this flag.
+      --kv.config string         Specifies TiKV configuration files, for development and debugging. You can ignore this flag.
+      --monitor                  Determines whether to start the monitor.
+      --pd int                   Specifies the number of PD instances in the cluster, 1 by default.
+      --pd.binpath string        Specifies the location of PD binary files, for development and debugging. You can ignore this flag.
+      --pd.config string         Specifies PD configuration files, for development and debugging. You can ignore this flag.
+      --tiflash int              Specifies the number of TiFlash instances in the cluster, 0 by default.
+      --tiflash.binpath string   Specifies the location of TiFlash binary files, for development and debugging. You can ignore this flag.
+      --tiflash.config string    Specifies TiFlash configuration files, for development and debugging. You can ignore this flag.
```

Comment:
```
Flags:
      --db int                   TiDB instance number (default 1)
      --db.binpath string        TiDB instance binary path
      --db.config string         TiDB instance configuration file
  -h, --help                     help for tiup
      --host string              Playground cluster host (default "127.0.0.1")
      --kv int                   TiKV instance number (default 1)
      --kv.binpath string        TiKV instance binary path
      --kv.config string         TiKV instance configuration file
      --monitor                  Start prometheus component
      --pd int                   PD instance number (default 1)
      --pd.binpath string        PD instance binary path
      --pd.config string         PD instance configuration file
      --tiflash int              TiFlash instance number
      --tiflash.binpath string   TiFlash instance binary path
      --tiflash.config string    TiFlash instance configuration file
```
The actual flag description is as above. We need to make the document consistent with what users actually see in the command-line tool. @TomShawn